easybuild/easyconfigs/x/Xmipp/library_imports.patch

```diff
@@ -0,0 +1,100 @@
+diff -ru xmipp/libraries/data/metadata_sql.h xmipp_working/libraries/data/metadata_sql.h
```

Comment:
include comments: why is this patch needed, where did you find it (or add yourself as an author)